packages/ui/src/components/molecules/ProductPrice/styles.scss

```diff
@@ -0,0 +1,15 @@
+[data-fs-product-price] {
+  // --------------------------------------------------------
+  // Design Tokens for Product Price
+  // --------------------------------------------------------
+
+  --fs-product-price-gap                     : var(--fs-spacing-1);
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
+
+  display: flex;
```

Comment:
@renatamottam  I decided to "extract" this part. Actually it will be duplicated since I haven't removed from the other components that use it. WDYT?